src/NetAnalyzers/Core/Microsoft.CodeQuality.Analyzers/Maintainability/UseNameofInPlaceOfString.cs

```diff
@@ -53,9 +53,9 @@ private void AnalyzeArgument(OperationAnalysisContext context)
             }
 
             var argument = (IArgumentOperation)context.Operation;
-            if ((argument.Value.Kind != OperationKind.Literal
+            if (argument.Value.Kind != OperationKind.Literal
                 || argument.ArgumentKind != ArgumentKind.Explicit
-                || argument.Value.Type.SpecialType != SpecialType.System_String))
+                || argument.Value.Type.SpecialType != SpecialType.System_String)
```

Comment:
I don't know if StyleCop can check that but it'd be nice to have the OR/AND always on the same side (left or right).

Reply:
The option is `dotnet_style_operator_placement_when_wrapping`